### PR TITLE
Adding lazypredict

### PR DESCRIPTION
**Pull request by shankarpandala**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by shankarpandala** [comment]
@conda-forge-admin, please relint

File: recipes/lazypredict/LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+MIT License
```
Comment (synapticarbors) [inline recipes/lazypredict/LICENSE:1]: The license file is included in the source distribution so no need to manually include it here. Please remove this file.

File: recipes/lazypredict/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+build:
+  number: 0
+  entry_points:
+    - lazypredict=lazypredict.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - click >=7.0
```
Comment (synapticarbors) [inline recipes/lazypredict/meta.yaml:20]: I believe you can remove `click` from the `host` section. You will need to add `pytest-runner`.

File: recipes/lazypredict/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+test:
+  imports:
+    - lazypredict
+  commands:
+    - pip check
+    - lazypredict --help
+  requires:
+    - pip
+    - pytest >=3
```
Comment (synapticarbors) [inline recipes/lazypredict/meta.yaml:35]: I don't think you need `pytest` here since you aren't running any tests from the test suite.

**Pull request closed without merging**